03-event-emitters/3-task/index.ts

```diff
@@ -0,0 +1,14 @@
+import CSVReader from "./csv-reader";
+import JSONWriter from "./json-writer";
+
+(async () => {
+  try {
+    const csvReader = new CSVReader("./3-task/csv/data.csv");
+    const jsonWriter = new JSONWriter("./3-task/json/data.txt");
+
+    const data = await csvReader.read();
+    await jsonWriter.write(data as string[]);
```

Comment:
If you use pipes, you can simplify this like so: 
![image](https://github.com/molek95/nodejs-global-mentoring/assets/39033848/2b2c7dc4-a28a-4eac-b2aa-2ceb81c98f20)